src/components/FilterListItem/FilterListItem.jsx

```diff
@@ -21,15 +21,15 @@ class FilterListItem extends React.Component {
               className="button is-link"
               title="Edit filter"
             >
-              <span className="octicon octicon-pencil"></span>
+              ✏️
             </button>
             <button
               onClick={() => this.props.delete(this.props.filter)}
               type="button"
               className="button is-link"
               title="Delete filter"
             >
-              <span className="octicon octicon-trashcan"></span>
+              ❌
```

Comment:
Watch it: I think this is the emoji used for 'ignore' on the main screen. I don't necessarily like it for ignore, but I think they should be different for delete v. ignore.